subsys/bluetooth/mesh/foundation.h

```diff
@@ -112,12 +112,25 @@
 #define STATUS_UNSPECIFIED                 0x10
 #define STATUS_INVALID_BINDING             0x11
 
+enum label_flag {
```

Comment:
I don't think you necessarily need a name for this enum, at least I don't think I saw you use the name anywhere